src/config/Extensibility/Middleware/middlewarers/generateMiddlewaredClass.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable @scandipwa/scandipwa-guidelines/export-level-one */
+const getWrapperFromPlugin = require('../helpers/getWrapperFromPlugin');
+const sortPlugins = require('../helpers/sortPlugins');
+
+/**
+ * Provide an opportunity to wrap proxy with additional functions.
+ * @param {Function} origMember
+ * @param {Array} sortedPlugins
+ * @param Context origContext
+ */
+module.exports = (proxy) => {
+    const { __namespace__ } = Object.getPrototypeOf(proxy);
+    const namespacePluginsClass = globalThis.plugins?.[__namespace__]?.['class'] || [];
```

Comment:
Why keep the name `__namespace__` as the variable, rename to `namespace`, otherwise, the:

```
globalThis.plugins?.[__namespace__]?.['class']
```

looks like a monstrosity.

Reply:
Instead of `.['class']` type `.class`